media/themes/Lautus/lautus.msstyles/lautus.rc

```diff
@@ -92,7 +92,21 @@ NORMAL_SPINDOWNGLYPH_BMP BITMAP "bitmaps/NORMAL_SPINDOWNGLYPH.bmp"
 NORMAL_SPINLEFTGLYPH_BMP BITMAP "bitmaps/NORMAL_SPINLEFTGLYPH.bmp"
 NORMAL_SPINRIGHTGLYPH_BMP BITMAP "bitmaps/NORMAL_SPINRIGHTGLYPH.bmp"
 NORMAL_SPINUPGLYPH_BMP BITMAP "bitmaps/NORMAL_SPINUPGLYPH.bmp"
+#ifdef __REACTOS__
```

Comment:
I guess `MSSTYLES_ACCEPT_PNG_IMAGES` can't be used here, can it?
Would creating the define in `CMakeLists.txt` help?